docs/dev/api/api-testing.md

```diff
@@ -829,7 +735,7 @@ Runs all tests in a project matching a tag.
   <tbody>
     <tr>
      <td><code>buildId</code></td>
-     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If this execution is associated to a build, the ID of the build should be provided, so that all resources can be associated to it.</p></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If this execution is associated with a build, the ID of the build should be provided so that all resources can be associated to it.</p></td>
```

Comment:
could we use "request" instead of "execution" 
```suggestion
     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If this execution is associated with a build, provide the build ID so that all resources can be associated to it.</p></td>
```